source/MaterialXContrib/MaterialXNode/Util.cpp

```diff
@@ -57,5 +42,24 @@ void loadLibraries(const MaterialX::StringVec& libraryNames,
 	}
 }
 
-} // namespace MAYA_MATERIALX_UTILS
+mx::FilePath findInSubdirectories(const mx::FileSearchPath& searchPaths,
```

Comment:
New tree search util. May migrate this to core as a utility. In fact all of these could be moved back. But TBD.